shell/src/main/java/org/apache/accumulo/shell/Shell.java

```diff
@@ -1263,4 +1263,20 @@ public boolean hasExited() {
     return exit;
   }
 
+  /**
+   * Prompt user for yes/no using the shell prompt.
+   *
+   * @param prompt the string printed to user, with (yes|no)? appended as the prompt.
+   * @return true if user enters y | yes.
+   */
+  public boolean yorn(final String prompt) {
```

Comment:
"yorn" is a weird name. "yesno" is more common for this sort of thing. Here's [an example in Lua](https://dev.fandom.com/wiki/Global_Lua_Modules/Yesno). "confirm" is also a good option.

Reply:
Switched to `confirm` as `yorn` is also very similar to `yarn`